### DISCUSSION TITLE
Making a standalone CSV package.

### DISCUSSION THREAD

**Discussion by gfissore-rai** [RelationalAI/sales-eng-tools#4] [Ideas]
The `sales-eng-tools` repo provides different "libraries" that can be imported by copy-pasting the relevant model code into a Rel codebase. With the introduction of an official package manager (hopefully coming soon), it would make sense to separate these libraries into their own packages.

A good starting point would be to make a standalone CSV package, as this can be useful in a multitude of very different Rel projects.

@grigory93 happy to hear your thoughts on this.

**Comment by grigory93** [comment]
Yes, providing themed functions (like load csv) as a standard Rel package makes sense. As soon as this functionality becomes available there are probably a few feasible packages that may branch out of the se_lib: and csv could be top one.
